app/controllers/companies_controller.rb

```diff
@@ -1,2 +1,38 @@
 class CompaniesController < ApplicationController
+  def index
+    @companies = Company.all
+    render html: "TODO: company display page"
```

Comment:
Perhaps considering doing the index page also as part of this PR?

Reply:
agree, should just do it in this PR if possible

Reply:
I have an idea on how to implement the index page after working on the autocompletion. If this work hasn't been done yet, I can do this work on a separate branch after this is merged.